automod/assets/automod.html

```diff
@@ -146,8 +146,8 @@ <h4>Create a new rule</h4>
                             <p>Advanced Automoderator (also known as Automoderator V2 or AMV2 ) is a completely new automoderator system made with the goal to be the most flexible, configurable system you could get for a chat bot (within reason).<br>
                                 You start by creating a ruleset, then creating rules inside said ruleset.<br>
                                 Rules consist of one or more triggers, conditions, and effects, you can also apply general conditions on the ruleset itself.<br>
-                                Visit the <a href="https://docs.yagpdb.xyz/tools-and-utilities/automoderator-v2" target="_blank">documentation</a> for more information on how to build rules and some examples.</p>
-                            <p>If you have some suggestions for triggers, conditions, or effects then either open a issue on <a href="https://github.com/botlabs-gg/yagpdb" target="_blank">GitHub</a> or join the support server (link located above).</p>
+                                Visit the <a href="https://help.yagpdb.xyz/docs/moderation/advanced-automoderator" target="_blank">documentation</a> for more information on how to build rules and some examples.</p>
+                            <p>If you have some suggestions for triggers, conditions,  or join the support server (link located above) and add a suggestion.</p>
```

Comment:
```suggestion
                            <p>If you have some suggestions for triggers, conditions, or join the support server (link located above) and add a suggestion.</p>
```

Reply:
I feel like we should put a hyperlink on "support server", and remove the "link located above", or just remove "link located above". Specifically this note feels extremely redundant.